src/components/constants/url.js

```diff
@@ -1,24 +1,25 @@
 const BASE_URL = `${process.env.NEXT_PUBLIC_BASE_URL}`;
 export const SUBSCRIBE_TO_CHALLENGE_URL = `${BASE_URL}/challenges/subscribe`;
-export const TASKS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks/self`;
-export const USER_PROFILE_URL = 'https://my.realdevsquad.com/profile';
-export const HOME_URL = 'https://realdevsquad.com';
-export const WELCOME_URL = 'https://welcome.realdevsquad.com';
-export const EVENTS_URL = 'https://www.realdevsquad.com/events.html';
-export const CRYPTO_URL = 'https://crypto.realdevsquad.com';
-export const STATUS_URL = 'https://status.realdevsquad.com';
+export const MY_TASKS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tasks/self`;
+export const USER_PROFILE_URL = "https://my.realdevsquad.com/profile";
+export const HOME_URL = "https://realdevsquad.com";
+export const WELCOME_URL = "https://welcome.realdevsquad.com";
+export const EVENTS_URL = "https://www.realdevsquad.com/events.html";
+export const CRYPTO_URL = "https://crypto.realdevsquad.com";
+export const STATUS_URL = "https://status.realdevsquad.com";
 export const LOGIN_URL = `https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97&state=${STATUS_URL}`;
-export const MEMBERS_URL = 'https://members.realdevsquad.com';
+export const MEMBERS_URL = "https://members.realdevsquad.com";
 export const CHALLENGES_URL = `${BASE_URL}/challenges`;
 export const USER_SELF = `${BASE_URL}/users/self`;
-export const DEFAULT_AVATAR = '/Avatar.png';
-export const RDS_LOGO = '/RDSLogo.png';
-export const GITHUB_LOGO = '/github-white.png';
-export const SIGNUP_LINK = 'https://my.realdevsquad.com/signup';
-export const LOGOUT_URL = 'https://api.realdevsquad.com/auth/signout';
+export const DEFAULT_AVATAR = "/Avatar.png";
+export const RDS_LOGO = "/RDSLogo.png";
+export const GITHUB_LOGO = "/github-white.png";
+export const SIGNUP_LINK = "https://my.realdevsquad.com/signup";
+export const LOGOUT_URL = "https://api.realdevsquad.com/auth/signout";
 export const IDLE_USERS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/users/status?state=IDLE`;
 export const ITEM_BY_FILTER_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/items/filter`;
 export const ITEMS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/items`;
 export const ALL_TAGS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/tags`;
 export const ALL_LEVELS_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/levels`;
-export const ITEM_TYPES = { task: 'TASK' };
+export const ITEM_TYPES = { task: "TASK" };
+export const TASK_URL = "https://api.realdevsquad.com/tasks";
```

Comment:
can you use `process.env.NEXT_PUBLIC_BASE_URL` instead of `https://api.realdevsquad.com`

Reply:
I have updated it 👍🏻

Reply:
it is not updated yet

Reply:
Now I have pushed the updated changes 👍🏻